TombEngine/Objects/TR5/Entity/tr5_roman_statue.cpp

```diff
@@ -374,6 +356,12 @@ namespace TEN::Entities::Creatures::TR5
 			int deltaFrame;
 			bool unknown;
 
+			int R = 0;
```

Comment:
Why this block with `R, G, B, color` was added? It seems only `R` is used further, and it is 0 anyway, so why not pass 0 directly?

Reply:
fixed